website/docs/tutorials/07-testing/03-fuzzing/README.md

```diff
@@ -287,10 +287,6 @@ Each resource counter presents a summary of observed values across all test runs
 
 ## Adjusting fuzzing quality
 
-By default, Protostar tries to fail a test case within 100 examples.
-The default value is chosen to suit a workflow where the test will be part of a suite that is
-regularly executed locally or on a CI server,
-balancing total running time against the chance of missing a bug.
```

Comment:
Why is this removed? It's the only place in docs which says what's the default...

Reply:
right, sorry

Reply:
[revert adjusting fuzzing quality](https://github.com/software-mansion/protostar/pull/995/commits/1d12293ba0b243f635e23ddff1d9552d3dc25aa2)